config/application.rb

```diff
@@ -45,9 +45,17 @@ class Application < Rails::Application
     config.x.feedback_url = "https://www.smartsurvey.co.uk/s/04QYDH/"
     config.x.display_raw_responses = %w[enabled true].include?(ENV.fetch("DISPLAY_RAW_RESPONSES", nil))
     config.action_mailer.deliver_later_queue_name = :mailers
+
+    # CDAPI
     config.x.court_data_api_config.uri = ENV.fetch("COURT_DATA_API_URL", nil)
     config.x.court_data_api_config.user = ENV.fetch("COURT_DATA_API_USERNAME", nil)
     config.x.court_data_api_config.secret = ENV.fetch("COURT_DATA_API_SECRET", nil)
+
+    # Direct connection to CDA V2
+    config.x.court_data_adaptor_api_v2_url = ENV.fetch("COURT_DATA_ADAPTOR_API_V2_URL", nil)
```

Comment:
You're not using these

Reply:
yep, there were used in my first implementation.